### ISSUE TITLE
get the content of DummyContent class out to a file

### ISSUE THREAD

**Issue by bacloud14** [bacloud14/Ideas#3]
For the developer to be able to edit and add new ideas to the catalogue, it would be better to use an resource json/xml/other file. 
In my opinion it would be great if possible to also keep DummyContent class and adapt it, so that we have two options. But it is ok "to just replace it totally with an external file".

**Comment by AliAkberAakash** [comment]
Hi I'd like to work on this issue. Would it be okay if I keep the data of DummyContent class in firebase and fetch it from there?

**Comment by bacloud14** [comment]
Hello Ali, 
Sûre ! Fantastic, you can work on it. 
Why would you like to put data I. Firebase, while this is very little data? I would prefer a file so that a user without connection would also benefit the app

**Comment by AliAkberAakash** [comment]
This is very little data now, but by adding firebase users data on the app will be updated automatically each time new data is added. And firebase has a system to cache data. So if the users load it just one time initially they'll be able to access it even when not connected to a network.

**Comment by bacloud14** [comment]
Understood, it is dump that I didn't thought of that, like an update for a new little idea to be added (my file cache like option haha)
Sure then, so we need a firebase account right ?

**Comment by AliAkberAakash** [comment]
Yes, I'd suggest u add firebase to the project,add necessary dependencies and the goole.json file. then I'll add code to read and write from firebase. It might me good If I'm given access to the firebase project for debugging. Also, in the rules section for firebase firestore, make read and write public, as we want users to read without creating an account.

**Comment by bacloud14** [comment]
Sure, w'll try to do this Saturday . thanks for your patience.
